### PR TITLE
Admin APIs

### PR DESCRIPTION
**Pull request by mywayisone**
Admin model, controller, and APIs created

### CODE REVIEW

File: pkg/config/app.go [hunk 1]
```diff
@@ -10,8 +10,8 @@ var (
 )
 
 func Connect() {
-	// dsn := "root:password@tcp(127.0.0.1:3306)/karygo?charset=utf8mb4&parseTime=True&loc=Local"
-	dsn := "ba15d8eef097fd:bc99a3bc@tcp(us-cdbr-east-06.cleardb.net:3306)/heroku_60b5ede111b594a?charset=utf8mb4&parseTime=True&loc=Local"
+	dsn := "root@tcp(127.0.0.1:3306)/karygo?charset=utf8mb4&parseTime=True&loc=Local"
```
Comment (imchukwu) [inline pkg/config/app.go]: when pushing switch back to the heroku database config, else it will break online

File: pkg/routes/admin-route.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	"github.com/imchukwu/karygo_backend/pkg/controllers"
+)
+
+var RegisterAdminRoutes = func (r *mux.Router){
+	r.HandleFunc("/admin", controllers.CreateAdmin).Methods("POST")
+	r.HandleFunc("/admin", controllers.GetAdmins).Methods("GET")
+	r.HandleFunc("/admin/{adminId}", controllers.GetAdmin).Methods("GET")
+	r.HandleFunc("/admin/{adminId}", controllers.UpdateAdmin).Methods("PUT")
+	r.HandleFunc("/admin/{admin}", controllers.DeleteAdmin).Methods("DELETE")
+	r.HandleFunc("/admin/transactiion", controllers.GetTransactions).Methods("GET")
```
Comment (imchukwu) [inline pkg/routes/admin-route.go:14]: i dont think we need the transaction route, please justify

**Review by imchukwu — CHANGES_REQUESTED** [review]

File: pkg/config/app.go (hunk 1 above)
Comment (mywayisone) [reply pkg/config/app.go]: Noted

File: pkg/routes/admin-route.go (hunk 2 above)
Comment (mywayisone) [reply pkg/routes/admin-route.go:14]: I think it is needed for review and clarity especially when there are issues with trips and items sent.

**Review by mywayisone — COMMENTED** [review]
I guess there is no more issue with Admin API.

**Review by mywayisone — COMMENTED** [review]
Corrections noted

**Pull request merged**